src/configure-webpack/loaders/javascript.js

```diff
@@ -83,10 +83,16 @@ export default {
           {
             test: fileExtensions.test.JAVASCRIPT,
             enforce: 'pre',
-            loader: 'eslint-loader',
+            loader: 'standard-loader',
             exclude: /node_modules/,
             options: {
-              configFile: path.join(projectPath, '.eslintrc')
+              env: {
+                jasmine: true
+              },
+              error: action !== actions.DEVELOP && action !== actions.TEST_UNIT,
+              parser: 'babel-eslint',
+              plugins: ['flowtype'],
```

Comment:
I can't find if the `standard-loader` supports `plugins` config key in the source code https://github.com/timoxley/standard-loader. Are you sure that we should have this here?